.travis.yml

```diff
@@ -2,7 +2,7 @@
 after_script:
 - rake travis:after -t
 before_script:
-- gem install hoe-travis --no-document
+- gem install hoe-travis hoe-bundler --no-document
```

Comment:
Is adding this necessary?

Plugins should be installed automatically via the `travis:before` task [like this](https://travis-ci.org/drbrain/net-http-persistent/jobs/529405593#L500), can you link me to a build where this failed?